### PR TITLE
Feature/transition end mode

### PR DESCRIPTION
**Pull request by titouanmathis**
### Added

- Add a `$parent` instance property (6b5088d)
- Add the ability to set the `$options` instance property (cf05aa3)
- Add a `jsconfig.json` file (e9d1e8b)
- Add an `.nvmrc` file (149aebd)

#### `utils/css/transition` (f98ba1c)

This PR introduces a new `endMode` parameter to the `transition` method:

- when set to `remove` (default), it will remove the latest styles or classes applied to the element at the end of the transition
- when set to `keep`, the latest styles or classes will be kept at the end of the transition

See the [updated doc](https://deploy-preview-26--studiometa-js-toolkit.netlify.app/utils/#transition) for more details.

#### `components/AccordionItem` (8f4350c)

A new `isOpen` option has beend added to the AccordionItem component, allowing it to be either open or closed in its initial state.

```html
<div data-component="AccordionItem" data-options='{ "isOpen": true }'>
  ...
</div>
```

See the [updated demo](https://deploy-preview-26--studiometa-js-toolkit-demo.netlify.app/) for an example.

### Changed

- Improve the `Accordion` component's accessibility (a786238)
- The package components now use the new `endMode` parameter (551c4ac)
- Update the NPM dependencies (29b6754)
- Update tests (3faff4d)
- Update documentation (b8ef01f)

### CODE REVIEW

**Comment by lusimeon** [comment]
@titouanmathis There is a typo in [this commit message](https://github.com/studiometa/js-toolkit/pull/26/commits/cad7822799c46443fee84583598967d0bf0b48bc).

File: docs/utils/readme.md [hunk 1]
```diff
@@ -162,3 +140,58 @@ const throttledFn = throttle(() => {
...
+// - opacity-0
+// - transition duration-500
+// - opacity-50
+transition(document.body, {
+  from: 'opacity-0',
+  active: 'transition duration-500',
+  to: 'opacity-50',
+});
+
+// Will apply and remove the following classes sequentially:
```
Comment (perruche) [inline docs/utils/readme.md:187]: Will apply and **keep** the following classes sequentially:

**Comment by titouanmathis** [comment]
> @titouanmathis There is a typo in [this commit message](https://github.com/studiometa/js-toolkit/pull/26/commits/cad7822799c46443fee84583598967d0bf0b48bc).

Nice catch, thanks!

File: docs/utils/readme.md (hunk 1 above)
Comment (titouanmathis) [reply docs/utils/readme.md:187]: Fixed, thanks!

**Comment by titouanmathis** [comment]
> Could we add documentation for the Accordion & AccordionItem Component while we have a PR extending the component API ?

[Done](https://deploy-preview-26--studiometa-js-toolkit.netlify.app/components/Accordion.html)! 

I also update some elements of the [API](https://deploy-preview-26--studiometa-js-toolkit.netlify.app/api/) documentation.

**Pull request merged**